src/dex-ui/DEX.tsx

```diff
@@ -13,22 +14,24 @@ const DEX = () => {
     <ChakraProvider theme={DEXTheme}>
       <Container color="black" w="100%" maxHeight="100%" maxWidth="100%" bg="#FFFFFF" marginBottom="5rem" centerContent>
         <Router>
-          <TopMenuBar menuOptions={menuOptions}></TopMenuBar>
-          <Flex width="66rem" justifyContent="center">
-            <Routes>
-              <Route path="/" element={<Navigate to="/swap" />} />
-              <Route path="/swap" element={<Swap />} />
-              <Route path="/pools" element={<Pools />} />
-              <Route path="/pools/add-liquidity" element={<AddLiquidity />} />
-              <Route path="/pools/withdraw" element={<Withdraw />} />
-              <Route path="/governance" element={<Governance />} />
-              <Route path="/governance/select-proposal-type" element={<SelectProposalType />} />
-              <Route path="/governance/select-proposal-type/new-token" element={<CreateProposal />} />
-              <Route path="/governance/select-proposal-type/text" element={<CreateProposal />} />
-              <Route path="/governance/select-proposal-type/token-transfer" element={<CreateProposal />} />
-              <Route path="/governance/select-proposal-type/contract-upgrade" element={<CreateProposal />} />
-            </Routes>
-          </Flex>
+          <ScrollToTop>
```

Comment:
The `ScrollToTop` component directs the user to the top of the destination page when routing to a new page.